go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/json-iterator/go v1.1.6 // indirect
 	github.com/kr/pretty v0.1.0 // indirect
+	github.com/markbates/inflect v1.0.4
```

Comment:
We no longer use this.
Now we use `github.com/gobuffalo/flect`.
I can't any place in your code that is using it. So drop this line and then run `go mod tidy`.